source/_docs/energy/battery.markdown

```diff
@@ -18,3 +18,13 @@ Some battery vendors have an API to integrate the data into your Home Assistant
 ### Using a CT clamp sensor
 
 {% include energy/ct_clamp.md %}
+
+Note: you have to create a 2 helpers to separate between "Energy going in to the battery" and "Energy coming out of the battery". This is best done by making template sensors with the following code (this one is for energy flowing in your battery):
+```
+{% if states('sensor.name_of_your_sensor')|float(0) >= 0 %}
+  {{ states('sensor.name_of_your_sensor') }}
+{% else %}
+  {{ 0 }}
+{% endif %}
```

Comment:
This is not correct; As this looks at a total of the two, and not the total of both import and export separated.

../Frenck